production/scrapers/california_vaccine_bi.R

```diff
@@ -0,0 +1,142 @@
+source("./R/generic_scraper.R")
+source("./R/utilities.R")
+
+california_vaccine_bi_pull <- function(x, wait = 10){
+    # scrape from the power bi iframe directly
+    y <- "https://app.powerbigov.us/view?r=" %>% 
+        str_c(
+            "eyJrIjoiODBjZjExNDktYWUxNi00NmM1LTllODMtY2VkMDM1MjlkODRiIiwidCI", 
+            "6IjA2NjI0NzdkLWZhMGMtNDU1Ni1hOGY1LWMzYmM2MmFhMGQ5YyJ9&", 
+            "pageName=ReportSection1d82f52cafdcc3e76847")
+    
+    remDr <- RSelenium::remoteDriver(
+        remoteServerAddr = "localhost",
+        port = 4445,
+        browserName = "firefox"
+    )
+    
+    del_ <- capture.output(remDr$open())
+    remDr$navigate(y)
+    
+    Sys.sleep(wait)
+    
+    xml2::read_html(remDr$getPageSource()[[1]])
+}
+
+get_california_vaccine_bi_table <- function(x, idx){
+    tab <- x %>%
+        rvest::html_nodes(".tableEx") %>%
+        # Assumes first table is incarcerated people, second is staff 
+        # Should check this in a better way 
+        .[idx] %>% 
+        rvest::html_node(".innerContainer")
+    
+    col_dat <- tab %>%
+        rvest::html_node(".bodyCells") %>%
+        rvest::html_node("div") %>%
+        rvest::html_children()
+    
+    # This is only returning the first 20 rows for some reason??  
+    dat_df <- do.call(rbind, lapply(col_dat, function(p){
+        sapply(rvest::html_children(p), function(z){
+            z %>% 
+                rvest::html_nodes("div") %>%
+                rvest::html_attr("title")})})) %>%
+        as.data.frame()
```

Comment:
Can't figure out why only the first 20 rows are being returned here!

Reply:
Looks like the first 20 rows of the table are in a different div than the rest of the table. The way that         `rvest::html_node("div")` works, it only grabs the FIRST node. I tested changing to this `rvest::html_nodes("div")`, and it still doesn't give it what we want. Anyways, the issue is stemming from `col_dat` not grabbing all of the divs we expect!

Reply:
I think the problem is in the pull function. If you look at the out put of save_raw in a browser there doesnt seem to be any elements which correspond to the rows at the end of each table. For example, facility PVSP is second to last in the prisoner table and sixth to last in the staff table and when you search the page directly in firefox for PVSP four hits come up. However if I search the raw html saved by the scraper there are no instances of the text pvsp. I think selenium is triggering the full table to load for some reason.
 
![2021-04-02-085516_1885x966_scrot](https://user-images.githubusercontent.com/5198764/113417609-c172cf80-9391-11eb-9642-6b26206cc5da.png)

Reply:
Threw in a hack for now where we save the html sorting by name for each table first ascending then descending. That way we can grab the facilities which fallin the first and last 20 positions alphabetically. This strategy only lest us get 40 rows so if CDCR adds more facilities we may miss them. I added a check in the code to see if we ever get 40 unique facilities which would indicate a problem.

Reply:
Thank you!! Added `Staff.Population` and retired the old CA vaccine and populations scrapers. I think it's good to merge, but I'll let you and @hjohns12 take another look!